apps/meteor/app/slashcommands-me/server/me.ts

```diff
@@ -10,7 +9,7 @@ import { slashCommands } from '../../utils/lib/slashCommand';
  */
 slashCommands.add(
 	'me',
-	function Me(_command: 'me', params: string, item: IMessage): void {
+	function Me(_command: 'me', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.